src/ui/popup/manage-settings-page/export-settings.tsx

```diff
@@ -0,0 +1,21 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import {Button} from '../../controls';
+
+export default function ExportButton(props: ViewProps) {
+    function exportSettings() {
+        const data = JSON.stringify(props.data.settings);
+        const a = document.createElement('a');
+        a.href = URL.createObjectURL(new Blob([data], {type: 'text/json'}));
+        a.download = 'DarkReader_Settings.json';
+        a.click();
+    }
+
+    return (
+        <Button
```

Comment:
Wrap `<Button>` into a `<ControlGroup>` and add description for those who don't understand what it does. Same for Import.